flink-streaming-connectors/flink-connector-elasticsearch2/src/test/java/org/apache/flink/streaming/connectors/elasticsearch2/ElasticsearchSinkITCase.java

```diff
@@ -210,6 +216,122 @@ public void cancel() {
 			running = false;
 		}
 	}
+	
+	@Test
+	public void testTemplateCreation() throws Exception {
+		// Settings.Builder settings=Settings.settingsBuilder().put("","");
+
+		Map<String, String> config = new HashMap<>();
+		// This instructs the sink to emit after every element, otherwise they
+		// would be buffered
+		config.put(ElasticsearchSink.CONFIG_KEY_BULK_FLUSH_MAX_ACTIONS, "1");
+		config.put("cluster.name", "my-transport-client-cluster");
```

Comment:
Try to see if "cluster.name" can be replaced by ClusterName.CLUSTER_NAME_SETTING